modules/installation-configuration-parameters.adoc

```diff
@@ -1036,6 +1044,40 @@ The GCP Compute Engine System service account email, like `<service_account_name
 |====
 
 endif::gcp[]
+ifdef::ibm-cloud[]
+[id="installation-configuration-parameters-additional-ibm-cloud_{context}"]
+== Additional IBM Cloud configuration parameters
+
+Additional IBM Cloud configuration parameters are described in the following table:
+
+.Additional IBM Cloud parameters
+[cols=".^1,.^6a,.^3a",options="header"]
+|====
+|Parameter|Description|Values
+
+|`platform.ibmcloud.resourceGroupName`
+|The name of an existing resource group to install your cluster to. This resource group must only be used for this specific cluster because the cluster components assume ownership of all of the resources in the resource group. If undefined, a new resource group is created for the cluster. ^1^
+|String, for example `existing_resource_group`.
+
+|`platform.ibmcloud.dedicatedHosts.profile`
+|The new dedicated host to create. If you specify a value for `platform.ibmcloud.dedicatedHosts.name`, this parameter is not required.
+|Valid IBM Cloud dedicated host profile, such as `cx2-host-152x304`. ^2^
```

Comment:
Looking at https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#alternative-footnote-styling-in-tables you might need brackets in the footnote callouts, but looks like it's rendering okay in the preview build. I would just double check this.

Reply:
Updated to include `[]`